src/migrations/201602231400_initial.js

```diff
@@ -22,17 +22,10 @@ function createFulfillmentsTable (knex) {
   })
 }
 
-function createEntryGroupsTable (knex) {
-  return knex.schema.createTableIfNotExists('entry_groups', table => {
-    table.increments()
-    table.timestamp('created_at').defaultTo(knex.fn.now())
-  })
-}
-
 function createEntriesTable (knex) {
   return knex.schema.createTableIfNotExists('entries', table => {
     table.increments()
-    table.integer('entry_group')
+    table.string('group', 1024)
```

Comment:
Nope, a single transfer can have 2 entry groups: one to apply debits, and another that either applies credits or reverts the debits.